pilot/pkg/security/authz/converter/converter.go

```diff
@@ -427,6 +432,9 @@ func convertAccessRuleToOperation(rule *authz_model.Permission) (*rbac_v1beta1.O
 	if rule == nil {
 		return nil, fmt.Errorf("invalid input: No rule found in ServiceRole")
 	}
+	if len(rule.Constraints) > 0 && len(rule.Constraints[0]) > 0 {
+		return nil, fmt.Errorf("serviceRole with constraints is not supported")
```

Comment:
Nit: ServiceRole should be capitalized.  The usual Go Rule doesn't apply [ https://github.com/golang/go/wiki/Errors#errors ] because ServiceRole is a proper name.  Use `// nolint: gas` to get around Lint.

Reply:
Done. PTAL